### PR TITLE
Updating expeditor configuration

### PR DESCRIPTION
**Pull request by jayashrig158**
Signed-off-by: jayashri garud <jgarud@msystechnologies.com>

i) The merge_actions subscription shortcut has been deprecated. All subscriptions should be declared via the subscriptions block for clarity.
ii) The promote block has been deprecated. All actions should be declared via the subscriptions block for clarity.
iii) 

[Please describe what this change achieves]

### Issues Resolved

[List any existing issues this PR resolves, or any Discourse or
StackOverflow discussions that are relevant]

### Check List

- [ ] New functionality includes tests
- [ ] All tests pass
- [ ] All commits have been signed-off for the Developer Certificate of Origin. See <https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco>

### CODE REVIEW

File: .expeditor/config.yml [hunk 1]
```diff
@@ -13,34 +13,24 @@ rubygems:
 release_branch:
   - ffi
 
-github:
```
Comment (tduffield) [inline .expeditor/config.yml:16]: This `github` section needs to remain

File: .expeditor/config.yml [hunk 2]
```diff
@@ -13,34 +13,24 @@ rubygems:
...
+      - bash:.expeditor/update_version.sh:
+          only_if: built_in:bump_version
+      - built_in:update_changelog:
+          ignore_labels:
+            - "Expeditor: Exclude From Changelog"
+            - "Expeditor: Skip All"
+      - built_in:build_gem:
+          only_if: built_in:bump_version
 
-changelog:
```
Comment (tduffield) [inline .expeditor/config.yml:25]: This `changelog` section needs to remain

**Review by tduffield — CHANGES_REQUESTED** [review]

File: .expeditor/config.yml (hunk 2 above)
Comment (skeshari12) [reply .expeditor/config.yml:25]: I have added that

**Review by tduffield — APPROVED** [review]

**Comment by tduffield** [comment]
Sorry, since `ffi` is the "default" branch, can we re-target the PR against that branch?

File: .expeditor/config.yml (hunk 1 above)
Comment (skeshari12) [reply .expeditor/config.yml]: Added that

File: .expeditor/config.yml [hunk 3]
```diff
@@ -10,6 +10,9 @@ slack:
 rubygems:
   - win32-mmap
 
+release_branch:
```
Comment (tduffield) [inline .expeditor/config.yml:13]: This should be `release_branches`

Comment (skeshari12) [reply .expeditor/config.yml:13]: Okay

**Pull request merged**